templates/site/js/dataproduct-view.js

```diff
@@ -0,0 +1,145 @@
+                var userCell = document.createElement("td");
+                var titleCell = document.createElement("td");
+                var linkAnchor = document.createElement("a");
+                var userCellSpan = document.createElement("span");
+
+                userCellSpan.classList.add("body-1");
+                linkAnchor.classList.add("body-1", "a-text-style");
+
+                userCellSpan.innerText = request.user;
+                linkAnchor.href = dataRequestViewURL + request.id;
+                linkAnchor.innerText = request.title;
+
+                userCell.appendChild(userCellSpan);
+                titleCell.appendChild(linkAnchor);
+
+                dataRequestRow.appendChild(userCell);
+                dataRequestRow.appendChild(titleCell);
+
+                dataRequestTableBody.appendChild(dataRequestRow);
+            }
+
+            if(marbleResult.links){
+                var nextLink;
+                var previousLink;
+
+                for(link of marbleResult.links){
+                    if(link.rel === "prev"){
+                        previousLink = link.href;
+                        previousLinkButton = document.getElementById("dataRequestPrevious");
+                        var oldPreviousButton = previousLinkButton;
+                        var newPreviousButton = oldPreviousButton.cloneNode(true);
+
+                        newPreviousButton.addEventListener("click", function(){
+                            getUploadedDataProducts(previousLink);
+                        });
+
+                        newPreviousButton.removeAttribute("disabled");
+                        newPreviousButton.classList.remove("disabled");
+
+                        oldPreviousButton.parentNode.replaceChild(newPreviousButton, oldPreviousButton);
+                    }
+
+                    if(link.rel === "next"){
+                        nextLink = link.href
+                        nextLinkButton = document.getElementById("dataRequestNext");
+                        var oldNextButton = nextLinkButton;
+                        var newNextButton = oldNextButton.cloneNode(true);
+
+                        newNextButton.addEventListener("click", function(){
+                            getUploadedDataProducts(nextLink);
+                        });
+
+                        newNextButton.removeAttribute("disabled");
+                        newNextButton.classList.remove("disabled");
+
+                        oldNextButton.parentNode.replaceChild(newNextButton, oldNextButton);
+                    }
+                }
+            }
+        }
+        else{
+            errorMessage.innerHTML = "Server error <br>  Data Products cannot be retrieved";
+
+            errorMessageContainer.classList.remove("display-none");
+            errorMessageContainer.classList.add("display-flex");
+
+            previousLinkButton.setAttribute("disabled", "disabled");
```

Comment:
This is no longer needed now that you're setting this at the beginning of this function